README.md

```diff
@@ -155,6 +170,40 @@ q.exact("rs2032651", "D"); // will only match genotype D
 q.exact("rs2032651", "AT"); // will only match genotype AT
 ```
 
+#### export()
```

Comment:
Yeah I thought abot this. However then it would not be possible to export custom functions like
```js
query.add(function(a,b) { return true })
query.export()
````
would not work.

Maybe we could also add a `toJSON` method that does strips the custom functions and formats it as a string?